include/n_menu/N_Menu.cpp

```diff
@@ -337,11 +334,11 @@ GMEXPORT double N_Menu_CreateToolWindow2(double height,double dragStyle){
 /*! \ingroup N_Menu
     Deletes a bitmap previously created with N_Menu_LoadBitmap. */
 GMEXPORT double N_Menu_DestroyBitmap(double bitmap){
-    for(unsigned int i = 0; i < numBitmaps; i++){
+    for(unsigned int i = 0; i < bitmaps.size(); i++){
         if(bitmaps[i] == (HBITMAP)(DWORD)bitmap){
             DeleteObject(bitmaps[i]);
             DrawMenuBar(hGmWnd);
-            bitmaps[i] = 0;
+            bitmaps.erase(bitmaps.cbegin()+i);

```

Comment:
This had me a little concerned until I realized this library works with pointers instead of list IDs as is typical of GM, so it should work fine.